### PR TITLE
Strings: Credit Card Validation

### PR DESCRIPTION
**Pull request by Bhargavishnu**
# Welcome to JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

Add credit card number string validator

### Describe your change:

* [x] Add an algorithm?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new JavaScript files are placed inside an existing directory.
* [x] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: String/ValidateCreditCard.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/**
```
Comment (raklaptudirm) [inline String/ValidateCreditCard.js:1]: Add references and a little explanation of how this works.

File: String/ValidateCreditCard.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * Validate a given credit card number
+ */
+
+const validStartSubString = ['4', '5', '6', '37', '34', '35']
```
Comment (raklaptudirm) [inline String/ValidateCreditCard.js:5]: Declare this inside the function.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]

File: String/ValidateCreditCard.js (hunk 1 above)
Comment (raklaptudirm) [reply String/ValidateCreditCard.js:1]: Also add a little description inside the comments, apart from the reference.

Comment (raklaptudirm) [reply String/ValidateCreditCard.js:1]: Uh, that is not what I meant. See [this](https://github.com/TheAlgorithms/Javascript/blob/master/Maths/FermatPrimalityTest.js) algorithm for example. They have the external reference, but also provide a short description of how the algorithm works.

Comment (Bhargavishnu) [reply String/ValidateCreditCard.js:1]: The code should be self explanatory. The reference and the single line comments should do it.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]
Please format your code with `standard.js`.

**Review by raklaptudirm — APPROVED** [review]

**Pull request merged**